.idea/misc.xml

```diff
@@ -5,26 +5,32 @@
     <option name="myDefaultNotNull" value="android.support.annotation.NonNull" />
     <option name="myNullables">
       <value>
-        <list size="7">
+        <list size="10">
```

Comment:
Definitely this one should be gitignored

Reply:
Done did it